app/models/user.rb

```diff
@@ -30,4 +30,9 @@ def refresh_token
   def get_picture
     social_providers.first.try(:profile_picture)
   end
+
+  def can_vote?
+    return true # if points >= 15
```

Comment:
Thanks for that observation. It was actually intentional, as nobody currently has any point. The comment will just be uncommented once we have this sorted out. :wink: